build.sbt

```diff
@@ -3,5 +3,5 @@ lazy val root = project
   .settings(
     name := "app-rpsosa",
     version := "1.0",
-    scalaVersion := "2.9.1"
+    scalaVersion := "2.12.3"
```

Comment:
Woops, I've missed this. You can use 2.12.6 which is the latest version, but 2.12.3 is just fine.

Reply:
I'll do it in the next step!